src/jqconsole.coffee

```diff
@@ -694,6 +698,7 @@ class JQConsole
     # (e.g. ">>> "), and the editable text to the left and right of the cursor.
     @$prompt_label = $(EMPTY_SPAN).appendTo @$prompt_current
     @$prompt_left = $(EMPTY_SPAN).appendTo @$prompt_current
+    @$prompt_left.attr 'class', CLASS_USER_TEXT
```

Comment:
Ah disregard this, I have checked GetTextPrompt() and the demo. It looks awesome!